external/llvm-project/mlir/include/mlir/Dialect/AMDGPU/IR/AMDGPUDialect.h

```diff
@@ -14,6 +14,7 @@
 #ifndef MLIR_DIALECT_AMDGPU_IR_AMDGPUDIALECT_H_
 #define MLIR_DIALECT_AMDGPU_IR_AMDGPUDIALECT_H_
 
+#include "mlir/Bytecode/BytecodeOpInterface.h"
```

Comment:
Where's this coming from?

Reply:
This is upstream : https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir/Dialect/GPU/IR/GPUDialect.h